src/inc/clrnt.h

```diff
@@ -789,6 +789,8 @@ typedef struct _DYNAMIC_FUNCTION_TABLE {
 //
 #ifdef _TARGET_AMD64_
 
+#define COMPACT_RUNTIME_FUNCTION        0
```

Comment:
> I tried with the following implementation of RtlpGetFunctionEndAddress

This implementation of RtlpGetFunctionEndAddress specific to the current ARM unwind info format. That's why it does not just work on x86.